local_setup.md

```diff
@@ -0,0 +1,193 @@
+# Local Setup
+
+For developing you want to set up the renuo-cms and the renuo-upload locally. They must be set up once and can then be used by every application that uses one or both of those services.
+
+## Step by Step Manual
+
+### Install renuo-cms-api
+
+Clone git repository and run bin/setup:
+
+```
+git clone git@github.com:renuo/renuo-cms-api.git
+cd renuo-cms-api
+bin/setup
+```
+
+If you already cloned the project, you can just pull the newest version and migrate the database.
+
+### Install renuo-upload signing
+
+Clone git repository and run bin/setup:
+
+```
+git clone git@github.com:renuo/renuo-upload-signing.git
+cd renuo-upload-signing
+bin/setup
+```
+
+If you already cloned the project, you can just pull the newest version and migrate the database.
+
+### Set Up a Amazon S3 bucket
+
+1.  Log in to amazon
+
+1.  Navigate to S3 by clicking on the S3 icon:
+
+   ![S3-Icon](s3-icon.png)
+
+1.  Create new bucket by clicking on blue button on the top with text "create bucket":
+
+    ![create bucket](create-bucket-button.png)
+
+1.  Name it however you like. We request to do something like ```renuo-upload-<yourname>-development``` and choose to place where your server should be. We usually use Frankfurt.
```

Comment:
> Name it however you like

Give it a meaningful name.

> We request

we suggest

> renuo-upload-\<yourname\>-development

from https://docs.google.com/document/d/1nu1LKFzck87wQAbSdM5n7Gi2SCX8Fw-IfwIgsUMHC-A/edit#heading=h.hlze8ijk1jq6 : 

> Use \<project-name>-local-\<user>-\<rails_env> for local names which interact with online services (e.g. S3).